pkg/util/steps/condition_test.go

```diff
@@ -9,21 +9,29 @@ import (
 	"testing"
 )
 
+type teststruct struct{}
```

Comment:
This is introduced because the names of methods have "-fm" as suffix differently from functions, isn't it?
It would be good to have the comment here mentioning that.

Reply:
done